apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue

```diff
@@ -77,6 +83,7 @@ const handleChangeDefaultGroupBy = async (selectedItems: GroupBySelectButtonItem
         const addedGroupByName: string = xor(costAnalysisPageState.groupBy, selectedItems.map((d) => d.name))[0];
         costAnalysisPageStore.$patch((_state) => {
             _state.groupBy = [addedGroupByName, ..._state.groupBy];
+            _state.chartGroupBy = addedGroupByName;
```

Comment:
question: what is chartGroupBy?

Reply:
![스크린샷 2023-09-11 오전 11 08 28](https://github.com/cloudforet-io/console/assets/18563857/bcedb41e-da64-4198-be95-611364eb2486)

This! chartGroupBy is only one.